spec.html

```diff
@@ -41552,6 +41554,8 @@ <h1>Additions and Changes That Introduce Incompatibilities with Prior Editions</
   <p><emu-xref href="#sec-regexp-pattern-flags"></emu-xref> In ECMAScript 2015, If the _pattern_ argument is a RegExp instance and the _flags_ argument is not *undefined*, a new RegExp instance is created just like _pattern_ except that _pattern_'s flags are replaced by the argument _flags_. In previous editions a *TypeError* exception was thrown when _pattern_ was a RegExp instance and _flags_ was not *undefined*.</p>
   <p><emu-xref href="#sec-properties-of-the-regexp-prototype-object"></emu-xref> In ECMAScript 2015, the RegExp prototype object is not a RegExp instance. In previous editions it was a RegExp instance whose pattern is the empty string.</p>
   <p><emu-xref href="#sec-properties-of-the-regexp-prototype-object"></emu-xref> In ECMAScript 2015, `source`, `global`, `ignoreCase`, and `multiline` are accessor properties defined on the RegExp prototype object. In previous editions they were data properties defined on RegExp instances.</p>
+  <p><emu-xref href="#sec-atomics.notify"></emu-xref> In ECMAScript 2019, `Atomics.notify` has been renamed `Atomics.wake` to prevent confusion with `Atomics.wait`</p>
```

Comment:
I think this is backwards, IIUC it was renamed *from* `Atomics.wake()` *to* `Atomics.notify()`.

Reply:
lol whoops, thanks, looked at git diff in the wrong order